src/appshell/internal/appshellconfiguration.cpp

```diff
@@ -147,9 +147,14 @@ std::string AppShellConfiguration::museScoreUrl() const
     return MUSESCORE_URL + languageCode;
 }
 
+std::string AppShellConfiguration::museScoreForumUrl() const
+{
+    return MUSESCORE_URL + "forum";
+}
+
 std::string AppShellConfiguration::museScoreContributionUrl() const
 {
-    return museScoreUrl() + "/donate";
+    return MUSESCORE_URL + "contribute";
```

Comment:
The leading slash is not needed, as `MUSESCORE_URL` has a trailing slash already. And it actually even harms, as it'd prevernt redirection to work (once that is put in place)